controls/radrichtextbox/features/selection.md

```diff
@@ -87,12 +236,13 @@ Here is an example of selecting each "RadRichTextBox" word in the text. This exa
 
 
 
-#### __VB.NET__
+#### **[VB.NET] Example 4: Select all occurrences of the "RadRichTextBox" word**
 
 {{region radrichtextbox-features-selection_3}}
+
 	Dim position As New DocumentPosition(Me.radRichTextBox.Document)
 	Do
-	 \'GetCurrentSpan().Text returns the word at the position
+	 'GetCurrentSpan().Text returns the word at the position
 	 Dim word As String = position.GetCurrentSpanBox().Text

```

Comment:
The comment is GetCurrentSpan, where the code contains GetCurrentSpanBox. 

Also, as I read the code sample, "GetCurrentSpan().Text returns the word at the position" statement is not always true. Should we remove this from the docs, and instead use MoveToCurrentWordStart/End if needed?